css/hw3.css

```diff
@@ -0,0 +1,77 @@
+* {
+    box-sizing: border-box;
+    font-size: 14px;
+}    
+.header-main{
+    text-align: center;
+    margin-bottom: 100px;
+}
+.header-list{
+    display: inline-block;
+    margin: 0px;
+    padding: 0px;
+}
+.header-item{
+    display: inline-block;
+    margin-right: 123px;
+    margin-top: 77px;
+}
+.header-item a{
+    text-decoration: none;
+    color: #626262;
+    text-transform: uppercase;
+    font-size: 14px;
+}
+.last-header-item{
```

Comment:
use https://developer.mozilla.org/ru/docs/Web/CSS/:last-child  remove this class